include/fluent-bit/flb_output.h

```diff
@@ -952,6 +960,102 @@ struct flb_output_flush *flb_output_flush_create(struct flb_task *task,
                 return NULL;
             }
 
+            out_flush->processed_event_chunk = flb_event_chunk_create(
+                                                evc->type,
+                                                0,
+                                                evc->tag,
+                                                flb_sds_len(evc->tag),
+                                                p_buf,
+                                                p_size);
+
+            if (out_flush->processed_event_chunk == NULL) {
+                flb_coro_destroy(coro);
+                flb_free(out_flush);
+                flb_free(p_buf);
+
+                return NULL;
+            }
+        }
+        else if (evc->type == FLB_EVENT_TYPE_PROFILES) {
+            p_buf = flb_calloc(evc->size * 2, sizeof(char));
+
```

Comment:
missing flb_errno();

Reply:
Addressed in [42f5d58](https://github.com/fluent/fluent-bit/pull/9748/commits/42f5d58e28867c391247fa73ea26302a1664f087)